releasenotes/README.md

```diff
@@ -26,12 +26,54 @@ Additional documentation:
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
 
-## 2.15.8 (Nov 1, 2023) [stable-rc]
+## 2.16.7 (Nov 8, 2023) [stable-rc]
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
 ------------
 
+* NEW: When creating a package version with the `package create version` command, you can now specify whether profile metadata components in a given package directory should be included in the new package. Set the new `scopeProfiles` option of the `sfdx-project.json` file to `true` in the configuration of a package directory for their profiles to be included; set it to `false` for them not to be included. If not specified, the default value of `scopeProfiles` is `false`. The `scopeProfiles` option is a child of `packageDirectory`, as shown in this example. (packaging PR [#416](https://github.com/forcedotcom/packaging/pull/416), schemas PR [#80](https://github.com/forcedotcom/schemas/pull/80), sfdx-core PR [#957](https://github.com/forcedotcom/sfdx-core/pull/957))
```

Comment:
I don't think this is worded quite right.  Setting `scopeProfiles: true` for the package directory means that profiles only from the package directory being packaged should be included and it will ignore any profile outside of that package.  With `scopeProfiles: false` (the default) it will include relevant pieces of profiles in any package directory defined in sfdx-project.json.